ui/raidboss/data/triggers/o11n.js

```diff
@@ -8,19 +8,22 @@
     {
       id: 'O11N Blaster',
       regex: /Blaster/,
+      regexFr: /Électrochoc/,
```

Comment:
You don't need to translate the regex for timeline triggers.  They're all based on the English timeline, even if the timeline is translated.